package.json

```
@@ -107,8 +110,8 @@
     "ts-toolbelt": "^8.0.6"
   },
   "peerDependencies": {
-    "boardgame.io": "^0.39.16",
-    "react": "^16"
+    "boardgame.io": "^0.39.16 || latest",
```

I think this range would skip versions 0.40–0.42, this would be better:

```suggestion
    "boardgame.io": ">=0.39.16",
```

NPM has this handy page where you can try out different version ranges and it will highlight what they mean in practice: https://semver.npmjs.com/

---

Thanks for the suggestion and link, and I agree! I'll change that

What about the react version? Is that range good, or would you prefer something else?

---

I think the React range is OK. It will include all 16.x and 17.x versions and it’s probably fair to work on the assumption that 18.x could introduce breaking changes.